components/aws/sagemaker/codebuild/scripts/deploy.sh

```diff
@@ -13,12 +13,13 @@
 
 set -e
 
-REMOTE_REPOSITORY="amazon/aws-sagemaker-kfp-components"
+REMOTE_REPOSITORY="public.ecr.aws/kubeflow-on-aws/aws-sagemaker-kfp-components"
 DRYRUN="true"
 FULL_VERSION_TAG=""
+BUILD_VERSION="v1"
```

Comment:
lets fail the build if version is not passed instead of default

Reply:
sure, helps avoid confusion/error in building v1/v2